### PR TITLE
Feature/billing reports

### PR DESCRIPTION
**Pull request by dee-dee-raye**
This really does work except for getting the price quote which breaks it. I believe this page itself is fine.  I tested it with a regular old BigDecimal and everything went well. I got printing to work too. The company performance report was a great example to work from!

### CODE REVIEW

File: src/acme/ui/ReportsPanel.java [hunk 1]
```diff
@@ -239,17 +259,146 @@ private void generateCompPerformanceReport(LocalDate from, LocalDate to) {
...
+        i = tickets.iterator();
+        count = 0;
+        onTime = 0;
+        totalPrice = 0;
+        while (i.hasNext()) {
+            t = (Ticket) i.next();
+          
+            
+            if(t.isBillToSender() && t.getPickupCustomer().getName().equals(customerName) || 
+                !t.isBillToSender() && t.getDeliveryCustomer().getName().equals(customerName))
```
Comment (simpleman19) [inline src/acme/ui/ReportsPanel.java]: Can you use UUID to determine if the customer is the same instead of name?

**Review by simpleman19 — CHANGES_REQUESTED** [review]
Also I am not seeing a report for any customer other than customer one but it may be fixed with the uuid change

**Comment by dee-dee-raye** [comment]
I could see all the customers last night. I walked through it just now and it's acting like there is only one ticket.

**Comment by dee-dee-raye** [comment]
It's working again for me. I don't know what changed. I just switched back to the branch and killed my db and reseeded it a couple of times.

**Comment by simpleman19** [comment]
I just ran it again and it appears to be working

**Review by simpleman19 — APPROVED** [review]

**Pull request merged**